tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/CallStackDebugTest.java

```diff
@@ -32,59 +33,63 @@
 import java.io.File;
 import java.nio.file.Paths;
 
+import static org.ballerinalang.debugger.test.utils.TestUtils.DebugResumeKind;
+import static org.ballerinalang.debugger.test.utils.TestUtils.testEntryFilePath;
+import static org.ballerinalang.debugger.test.utils.TestUtils.testProjectBaseDir;
+import static org.ballerinalang.debugger.test.utils.TestUtils.testProjectPath;
+
 /**
  * Test class for debug hit call stack representation.
  */
 public class CallStackDebugTest extends DebugAdapterBaseTestCase {
 
     @BeforeClass
     public void setup() throws BallerinaTestException {
-        testProjectName = "callstack-tests";
-        testModuleFileName = "main.bal";
+        String testProjectName = "callstack-tests";
+        String testModuleFileName = "main.bal";
         testProjectPath = testProjectBaseDir.toString() + File.separator + testProjectName;
         testEntryFilePath = Paths.get(testProjectPath, testModuleFileName).toString();
 
-        addBreakPoint(new BallerinaTestDebugPoint(testEntryFilePath, 35));
-        addBreakPoint(new BallerinaTestDebugPoint(testEntryFilePath, 44));
-        addBreakPoint(new BallerinaTestDebugPoint(testEntryFilePath, 40));
-        initDebugSession(DebugUtils.DebuggeeExecutionKind.RUN);
+        TestUtils.addBreakPoint(new BallerinaTestDebugPoint(testEntryFilePath, 35));
+        TestUtils.addBreakPoint(new BallerinaTestDebugPoint(testEntryFilePath, 44));
+        TestUtils.addBreakPoint(new BallerinaTestDebugPoint(testEntryFilePath, 40));
+        TestUtils.initDebugSession(DebugUtils.DebuggeeExecutionKind.RUN);
```

Comment:
Shall we import the static method instead of using qualified method calls everywhere? Check other places as well

Reply:
+1 for importing as static methods.

Reply:
I'm referring debug test utilities(`DebugTestRunner`) with reference variable. Therefore using `debugTestRunner` variable now to call the debug test utility methods.

Reply:
+1